docs/kubectl-moco.md

```diff
@@ -69,3 +69,9 @@ Fetch the credential information of a specified user
 ## `kubectl moco switchover CLUSTER_NAME`
 
 Switch the primary instance to one of the replicas.
+
```

Comment:
It would be nice to have a link to a page with detailed instructions on how to use it.
https://cybozu-go.github.io/moco/usage.html?highlight=stop#stop-clustering-and-reconciliation